conf.d/pure.fish

```diff
@@ -47,6 +47,11 @@ _pure_set_default pure_color_jobs pure_color_normal
 _pure_set_default pure_show_system_time false
 _pure_set_default pure_color_system_time pure_color_mute
 
+# Nix build environment
+_pure_set_default pure_enable_nixdevshell true
+_pure_set_default pure_symbol_nixdevshell_prefix ""
```

Comment:
As it will be disabled by default, you can propose a default prefix. I'm not familiar with Nix community so I trust you to pick a sensible one :+1:

Reply:
Done!